### ISSUE TITLE
Tweaks to 'All' clade filter button

### ISSUE THREAD

**Issue by PaulHeggarty** [lingdb/CoBL#243]
When the All button is clicked to turn off all clades, then it should become faded, just like the others.  Even if the user then does turn on individual clades by clicking those filter buttons, the All button should remain faded.  It should only go back to being 'on' when a user clicks on All specifically. (Ignore cases where the user manually turns all clades back on individually, no need to cater for that case.)

In short, the behaviour should be simple, and depend only on the All button itself.  When toggled on, to show all, then make it full colour.  When toggled off, then make it faded.
